app/view_models/nsm/v1/additional_fee.rb

```diff
@@ -15,6 +15,9 @@ class AdditionalFee < BaseWithAdjustments
       attribute :claimed_total_inc_vat
       attribute :assessed_total_inc_vat
 
+      adjustable_attribute :include_youth_court_fee, :boolean
+      attribute :youth_court_fee_adjustment_comment, :string
```

Comment:
So as I understand it, the goal hitherto has been to make this model agnostic about what _type_ of additional fee is being claimed, to make it more future-proof. Is there any way we can keep `youth_court`-specificness out  of this file?

Reply:
guess we could have a youth_court_fee view model that inherits this? then update everything else in line with the idea that we will have multiple different types of additional fee

Reply:
refactored youth_court_fee model to inherit from additional_fee